### PR TITLE
[patternfly] update site url and selectors

### PR DESCRIPTION
**Pull request by jschuler**
Updated some selectors and added synonyms

### CODE REVIEW

File: configs/patternfly.json [hunk 1]
```diff
@@ -13,30 +13,317 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "header nav [aria-current]",
+      "selector": "header nav li a",
```
Comment (s-pace) [inline configs/patternfly.json]: Why did you remove the `[aria-current]`selector? Globals selectors should only match one element from the page, the active one.

File: configs/patternfly.json [hunk 2]
```diff
@@ -13,30 +13,317 @@
...
       "global": true,
       "default_value": "Documentation"
     },
     "lvl1": {
-      "selector": "#page-sidebar [aria-expanded='true']",
+      "selector": "#page-sidebar li a",
```
Comment (s-pace) [inline configs/patternfly.json]: Same review as lvl0

File: configs/patternfly.json [hunk 3]
```diff
@@ -13,30 +13,317 @@
...
       "global": true,
       "default_value": "Chapter"
     },
-    "lvl2": "main h1",
-    "lvl3": "main h2",
-    "lvl4": "main h3",
-    "lvl5": "main h4",
-    "lvl6": "main h5",
+    "lvl2": {
+      "selector": "#page-sidebar section li a",
```
Comment (s-pace) [inline configs/patternfly.json]: Same review as lvl0

File: configs/patternfly.json [hunk 4]
```diff
@@ -13,30 +13,317 @@
...
   "conversation_id": [
     "772189694"
   ],
-  "nb_hits": 15754
+  "nb_hits": 15754,
+  "custom_settings": {
+    "separatorsToIndex": "_",
+    "synonyms": [
+      [
+        "about modal",
```
Comment (s-pace) [inline configs/patternfly.json]: The scraper supports only supports regular *one* word synonyms. Please remove the 2 words one.

File: configs/patternfly.json [hunk 5]
```diff
@@ -13,30 +13,317 @@
...
-    "separatorsToIndex": "_"
-  },
   "conversation_id": [
     "772189694"
   ],
-  "nb_hits": 15754
+  "nb_hits": 15754,
+  "custom_settings": {
+    "separatorsToIndex": "_",
+    "synonyms": [
```
Comment (s-pace) [inline configs/patternfly.json]: You can only have up to 20 synonyms if you need more details, you can [have a look here](https://community.algolia.com/docsearch/config-file.html#custom_settings-optional)

File: configs/patternfly.json (hunk 1 above)
Comment (jschuler) [reply configs/patternfly.json]: Thank you, I will revert this or at least ensure it targets just the active one

File: configs/patternfly.json (hunk 4 above)
Comment (jschuler) [reply configs/patternfly.json]: Okay, in the examples I saw something like `ECMAScript 6`, so that is also not valid, right?

File: configs/patternfly.json (hunk 5 above)
Comment (jschuler) [reply configs/patternfly.json]: 20 synonym arrays or each of the arrays is limited to 20 synonyms? Alternatively I can set algoliaOptions.synonyms with docsearch from the client?

File: configs/patternfly.json (hunk 4 above)
Comment (s-pace) [reply configs/patternfly.json]: Good point, will fix it. Thanks for pointing out.

File: configs/patternfly.json (hunk 5 above)
Comment (s-pace) [reply configs/patternfly.json]: The array of synonyms must be inferior or equals to 20. You can not edit synonym at the search time. Only at the indexing one.

**Comment by jschuler** [comment]
@s-pace Thank you, I have removed the custom synonyms for now, instead modifying the site url and the selectors

**Comment by s-pace** [comment]
@jschuler I have fine tuned the configuration to avoid extracting too many element from your page. Changes are deployed and live. Let us know if you want to change anything.
